frequency-plans.yml

```diff
@@ -159,6 +159,14 @@
   country-codes: [ar, au, bo, br, ca, cl, co, do, nz, py, pe, sr, uy]
   file: AU_915_928_FSB_8.yml
 
+- id: AU_915_928_FSB_1_NDT
+  band-id: AU_915_928
+  name: Australia 915-928 MHz, FSB 1
+  description: Frequency plan for Brazil (Australia base), using sub-band 1 and dwell time disabled
+  base-frequency: 915
+  country-codes: [br]
+  file: AU_915_928_FSB_1_NDT.yml
```

Comment:
Few requests based on the previous discussion:

- add plans for the other FSBs
- differentiate the `name` from the others (now they have the same name). Probably just append `for Brazil` or something
- use the override mentioned by @adriansmares